blocks/v2-images-grid/v2-images-grid.css

```diff
@@ -0,0 +1,95 @@
+:root {
+  --v2-images-grid-gutter: 8px;
+  --v2-images-grid-big-height: 168px;
+  --v2-images-grid-small-height: 125px;
+  --v2-images-grid-padding-space: 24px;
+}
+
+.redesign-v2 .v2-images-grid-container .v2-images-grid-wrapper {
+  padding: 0 100px;
+}
+
+.v2-images-grid-container .with-marker::before {
+  width: 37px;
+}
+
+.v2-images-grid-title {
```

Comment:
Can we use BEM for the classnames? 🙏🏻

Reply:
Not sure if the title should be part of the image grid or text block, but in case it is, the designs are using H2 font, not h1.

Reply:
> Can we use BEM for the classnames? 🙏🏻

Done

Reply:
> Not sure if the title should be part of the image grid or text block, but in case it is, the designs are using H2 font, not h1.

I have removed it as Tomas mentioned it's part cards block